packages/app/ui/components/BareChatInput/index.tsx

```diff
@@ -670,16 +673,33 @@ export default function BareChatInput({
         return;
       }
 
+      if (
+        keyEvent.key === 'ArrowUp' ||
+        (keyEvent.key === 'ArrowDown' && showMentionPopup)
```

Comment:
do we want the conditional to work like this? seems like it should be (up or down) and showMentionPopup, not up or (down and showMentionPopup)

Reply:
Yeah, that's definitely mistyped. Nice catch!